chapters/ru/chapter5/1.mdx

```diff
@@ -0,0 +1,17 @@
+# Введение
+
+В [главе 3] (/course/ru/chapter3) вы поверхностно ознакомились с библиотекой 🤗 Datasets и увидели три главных шага для использования ее в процессе fine-tuning: 
```

Comment:
Not sure if the `B` needs to be included in the link, but we need the space to be removed :)
```suggestion
В [главе 3](/course/ru/chapter3) вы поверхностно ознакомились с библиотекой 🤗 Datasets и увидели три главных шага для использования ее в процессе fine-tuning: 
```